fix: Fix storybook build command

## Description
Print error message
`Module not found: Error: Can't resolve 'financial-charts' in './src/templates/candlestickChart'`

## Steps to check or reproduce
Fix storybook build command

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -2,11 +2,10 @@
 name: Build and Publish Storybook to GitHub Pages
 
 on:
-  push
   # Event for the workflow to run on
-  # push:
-  # branches:
-  #   - "main" # Replace with the branch you want to deploy from
+  push:
+  branches:
+    - "main" # Replace with the branch you want to deploy from
 
 permissions:
   contents: read

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -30,7 +30,7 @@ jobs:
       #👇 Add Storybook build and deploy to GitHub Pages as a step in the workflow
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
         with:
-          install_command: npm ci # default: npm ci
+          install_command: npm i # default: npm ci
           build_command: npm run build:stories # default: npm run build-storybook
           path: /packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -27,10 +27,14 @@ jobs:
         with:
           node-version: "18.x"
 
-      #👇 Add Storybook build and deploy to GitHub Pages as a step in the workflow
-      - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
+      - name: Install dependencies
+        run: npm install --no-package-lock
+
+      - name: Build storybook
+        run: npm run build && npm run build:stories
+      
+      - name: Deploy to gh-pages branch
+        uses: peaceiris/actions-gh-pages@v3
         with:
-          install_command: npm install --no-package-lock # default: npm ci
-          build_command: npm run build && npm run build:stories # default: npm run build-storybook
-          path: packages/stories/storybook-static # default: dist/storybook
-          checkout: false # default: true
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: packages/stories/storybook-static

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -31,6 +31,6 @@ jobs:
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
         with:
           install_command: npm ci # default: npm ci
-          build_command: npm run build && npm run build:stories # default: npm run build-storybook
+          build_command: npm run build:stories # default: npm run build-storybook
           path: /packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -4,8 +4,8 @@ name: Build and Publish Storybook to GitHub Pages
 on:
   # Event for the workflow to run on
   push:
-  branches:
-    - "main" # Replace with the branch you want to deploy from
+    branches:
+      - "main" # Replace with the branch you want to deploy from
 
 permissions:
   contents: read

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -25,12 +25,12 @@ jobs:
       # Set up Node
       - uses: actions/setup-node@v3
         with:
-          node-version: "16.x"
+          node-version: "18.x"
 
       #👇 Add Storybook build and deploy to GitHub Pages as a step in the workflow
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
         with:
           install_command: npm i # default: npm ci
           build_command: npm run build:stories # default: npm run build-storybook
-          path: /packages/stories/storybook-static # default: dist/storybook
+          path: packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -31,6 +31,6 @@ jobs:
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
         with:
           install_command: npm install --no-package-lock # default: npm ci
-          build_command: npm run build:stories # default: npm run build-storybook
+          build_command: npm run build && npm run build:stories # default: npm run build-storybook
           path: packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -36,5 +36,5 @@ jobs:
       - name: Deploy to gh-pages branch
         uses: peaceiris/actions-gh-pages@v3
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ secrets.GH_PAGES_TOKEN }}
           publish_dir: packages/stories/storybook-static

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -2,10 +2,11 @@
 name: Build and Publish Storybook to GitHub Pages
 
 on:
+  push
   # Event for the workflow to run on
-  push:
-    branches:
-      - "main" # Replace with the branch you want to deploy from
+  # push:
+  # branches:
+  #   - "main" # Replace with the branch you want to deploy from
 
 permissions:
   contents: read
@@ -31,5 +32,5 @@ jobs:
         with:
           install_command: npm ci # default: npm ci
           build_command: npm run build && npm run build:stories # default: npm run build-storybook
-          path: packages/stories/storybook-static # default: dist/storybook
+          path: /packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/.github/workflows/gh-page.yml
+++ b/.github/workflows/gh-page.yml
@@ -30,7 +30,7 @@ jobs:
       #👇 Add Storybook build and deploy to GitHub Pages as a step in the workflow
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.1
         with:
-          install_command: npm i # default: npm ci
+          install_command: npm install --no-package-lock # default: npm ci
           build_command: npm run build:stories # default: npm run build-storybook
           path: packages/stories/storybook-static # default: dist/storybook
           checkout: false # default: true

--- a/packages/stories/package.json
+++ b/packages/stories/package.json
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "description": "Build financial-chart using javascript",
+  "homepage": "https://seungdeok.github.io/financial-charts/",
   "keywords": [
     "react",
     "javascript",

--- a/packages/stories/src/templates/lineChart/LineChart.tsx
+++ b/packages/stories/src/templates/lineChart/LineChart.tsx
@@ -1,5 +1,5 @@
 import React from "react";
-import { LineChart, LineChartProps } from "@financial-charts/templates";
+import { LineChart, LineChartProps } from "financial-charts";
 import { withAxis, PriceData } from "../../data/withAxis";
 import { DATA_SET } from "../../data/dummy";
 

